README.md

```diff
@@ -31,14 +69,393 @@ Permissions to allow the user to interact with the support center:
         }
     ]
 }
+
 ```
 
 
-## Testing your AWS account credentials with the CLI
+## 1.3. Workflow
+
+First, an AcccountPool must be created to specify the number of desired accounts to be ready. The operator then goes and creates that number of accounts. 
+When a [Hive](https://github.com/openshift/hive) cluster has a new cluster request, an AccountClaim is created with the desired name of the cluster in a unique namespace. The operator links the AccountClaim to an Account in the pool, and creates the required k8s secrets, placing them in the AccountClaim's unique namespace. The AccountPool is then filled up again by the operator.  Hive then uses the secrets to create the AWS resources for the new cluster. 
```

Comment:
Maybe this is a little more clear? "an AccountClaim is created with the name equal to the desired name of the cluster in a unique workspace" Also "and created the required k8s secrets containing the AWS credentials in the aws-account-operator namespace and the unique cluster namespace for hive to use during provisioning" 

Maybe with the rewording you can get rid of the last sentence?